doc/api_methods/AddSubscriberField.md

```diff
@@ -63,3 +63,25 @@ The common properties for all types:
 | created_at | string\|null | - | UTC time of creation in `Y-m-d H:i:s` format |
 | updated_at | string | - | UTC time of last update in `Y-m-d H:i:s` format |
 
+## Error handling
+
+All expected errors from the API are exceptions of class `\MailPoet\API\MP\v1\APIException`. 
+Code of the exception is populated to distinguish between different errors.
+
+An exception of base class `\Exception` can be thrown when something unexpected happens.
+
+Codes description:
+
+| Code | Description |
+| --- | ---  |
+| 1 | The subscriber couldn’t be created in the database |
+| 1001 | Missing a mandatory field in the `$data` argument |
```

Comment:
Why do these validation errors have a different numeration scheme? Shouldn't then all error codes be uniform (e.g. 1XXX - general errors, 2XXX - validation errors etc.)?

Reply:
No big reason really. It is just because I was adding those errors in a different file so I used a different scheme. Just to prevent future collisions.

I was thinking about your suggestion, but it is quite hard to separate the errors into different classes to be any useful so I didn't do that.